src/main/java/com/ericsson/eiffelcommons/utils/HttpRequest.java

```diff
@@ -264,6 +267,23 @@ public HttpRequest setBody(File file, ContentType contentType) throws IOExceptio
         return setBody(fileContent, contentType);
     }
 
+    /**
+     * Function that sets the Authorization header of the http request.
+     *
+     * @param httpRequest
+     * @param username
+     * @param password
+     * @return
+     * @throws UnsupportedEncodingException
+     */
+    public HttpRequest setCredentials(HttpRequest httpRequest, String username,
```

Comment:
Please remove httpRequest as parameter since this is the HttpRequest

Reply:
Done